build.go

```diff
@@ -37,6 +43,44 @@ const header = `# This config was built from JSON using NGINX crossplane.
 
 `
 
+type LuaBuild struct{}
```

Comment:
It might be nice to move the lua-specific types into a `lua.go` file to keep them organize together.

As per my other comments, please document these types/methods and maybe consider a single `Lua` struct that provide a builder and a lexer?

Reply:
done